Extension/src/LanguageServer/client.ts

```diff
@@ -2964,6 +2964,16 @@ export class DefaultClient implements Client {
         this.handleRemoveCodeAnalysisProblems(false, identifiersAndUris);
     }
 
+    private static countNewlines(text: string): number {
+        let numNewlines: number = 0;
+        for (let index: number = 0; index < text.length; ++index) {
```

Comment:
Or you can do it without a loop:
`return (text.match(/\n/g) || []).length;`

Reply:
Sure, that seems good to me.